### PR TITLE
chore(developer): add context/new-model-project-parameters

### PR DESCRIPTION
**Pull request by Nnyny**
partial fix of: https://github.com/keymanapp/keyman/issues/2131

@keymanapp-test-bot skip

### CODE REVIEW

**Comment by keyman-server** [comment]
This pull request is from an external repo and will not automatically be built. The build must still be passed before it can be merged. Ask one of the team members to make a manual build of this PR.

File: developer/src/tike/xml/help/contexthelp.xml [hunk 1]
```diff
@@ -387,6 +387,93 @@
...
+    <Control Name="editModelName" Title="Model Name">
+      <p>We recommend the name of the language, dialect, or community that this model is 
+      intended for. The name must be written in all the Latin letters or Arabic numerals.</p>
+    </Control>
+
+    <Control Name="editCopyright" Title="Copyright">
+      <p>Who owns the rights to this model and its data? Typically, 
+      you can use the automatically generated default value: © 2024 Your Full Name or 
+      Your Organization.</p>
+    </Control>
```
Comment (mcdurdin) [inline developer/src/tike/xml/help/contexthelp.xml:405]:
```suggestion
    <Control Name="editCopyright" Title="Copyright">
      <p>Who owns the rights to this model and its data? This field should
      contain the word "Copyright", the copyright symbol "©", and the full name of the rights 
      owner. Do not put the year of copyright in this field (see Full Copyright). Typically, 
      you can use the automatically generated default value: "Copyright © Your Full Name or 
      Your Organization".</p>
    </Control>
```

File: developer/src/tike/xml/help/contexthelp.xml [hunk 2]
```diff
@@ -387,6 +387,93 @@
...
+
+    <Control Name="editFullCopyright" Title="Full copyright">
+      <p></p>
```
Comment (mcdurdin) [inline developer/src/tike/xml/help/contexthelp.xml:408]:
```suggestion
      <p>Who owns the rights to this model and its data? This field should
      contain the word "Copyright", the copyright symbol "©", the first year of copyright, 
      and the full name of the rights owner. Do include the year of copyright in this field.
      Typically, you can use the automatically generated default value: 
      "Copyright © 2024 Your Full Name or Your Organization".</p>
```

**Review by mcdurdin — APPROVED** [review]
LGTM. Perhaps we can add the updated Copyright descriptions back to help.keyman.com 17.0 documentation?

**Comment by Nnyny** [comment]
Noted!

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 18.0.51-alpha](https://keyman.com/downloads/releases/alpha/18.0.51)

**Pull request merged**